test/extended/apiserver/apply.go

```diff
@@ -69,6 +70,12 @@ var _ = g.Describe("[sig-api-machinery][Feature:ServerSideApply] Server-Side App
 			// create the testing namespace
 			testNamespace := oc.SetupProject()
 
+			exist, err := exutil.DoesApiResourceExist(oc.AdminConfig(), gvr.Resource, gvr.GroupVersion().String())
+			o.Expect(err).NotTo(o.HaveOccurred())
+			if !exist {
```

Comment:
I'm not sure that blank skipping all non-existing API might be good approach, I'd prefer having an exception list which should be consulted and printed in the logs. This will ensure that in case of a problem we will actually catch it, rather than not be aware of it.

Reply:
The current exclusion of tests (where the test name contains explicit list of apigroups) is based on invocation of `DoesApiResourceExist` under the hood and skipping tests which has at least one API group missing. Which is equivalent to what's happening right here with the change.

> I'd prefer having an exception list which should be consulted and printed in the logs. 

Would you please more elaborate on how the list should look like?

Reply:
This came up a couple times.  If we want a test for "these things must exist", such a test can be created, but it should be distinct from "do the things that exist work".  That makes it easier to write logic here and easier to interpret failures when we see them because it's not testing a side-effect.

I think it's a good idea to create a test for, "this set of resources must exist" and using the microshift subset is a good starting point.